Binaries/DafnyRuntime.cs

```diff
@@ -1098,7 +1125,13 @@ public partial class Helpers {
       System.Console.Write(ToString(g));
     }
     public static G Default<G>() {
-      System.Type ty = typeof(G);
+      Type ty = typeof(G);
+      // TODO-RS: How to generalize this?
+      if (ty.Name.Equals("ISequence`1")) {
+        Type classType = Type.GetType("Dafny.Sequence`1");
```

Comment:
Yes this is super-sloppy and needs to be cleaned up. ``ISequence`1`` is shorthand for "The `ISequence` type that takes one type parameter" (since C# allows overloading by number of type parameters). It's just a little tricky because I need to map from `Dafny.ISequence<x>` to `Dafny.Sequence<x>`, where x is a concrete type rather than the generic type parameter `T`, and I'm still learning the C# reflection model. I know how to do it in Java! :)